Scripts/Mac/PrepFunctionalTests.sh

```diff
@@ -37,6 +37,8 @@ fi
 
 git-credential-manager install
 
+$VFS_SCRIPTDIR/InstallSharedDataQueueDylib.sh || exit 1
```

Comment:
Same here, rename the script to indicate it's installing a bug fix. This usage is currently unclear about _why_ it's happening.